tests/framework_tests/test_analytics.py

```diff
@@ -24,20 +27,20 @@ def test_get_total_activity_count(self):
         date = datetime.utcnow()
 
         assert_equal(analytics.get_total_activity_count(user._id), 0)
-        assert_equal(analytics.get_total_activity_count(user._id), user.get_activity_points(db=self.db))
+        assert_equal(analytics.get_total_activity_count(user._id), user.get_activity_points(db=None))
```

Comment:
Yeah, would be good to remove the unnecessary kwarg. I understand why you kept it though. It's up to you; it can be removed in a follow-up PR.